Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -260,6 +260,14 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+/** Set whether or not to log to stdout.
+ *
+ * @param enabled Whether or not stdout logging should be enabled.
+ *
+ * By default stdout logging is enabled.
+ */
+void kslog_setLogToStdout(bool enabled);
```

Comment:
Do we need to keep the `kscrash_setAddConsoleLogToReport` and `kscrash_setPrintPreviousLog` functions? They are plumbed up as properties on the KSCrash class as well. They are the ones using `kslog_setLogFilename` today. If we need to keep the behavior we need to introduce a way for the logger to log to a file or stdout, *and* a file used for the `addConsoleLogToReport` feature 😢